app/views/catalog/_downloads.html.erb

```diff
@@ -8,6 +8,8 @@
       <%= link_to(download_text(document.download_types.first[0]),
       download_hgl_path(id: document), data: {ajax_modal: 'trigger', download: 'trigger', download_type: 'harvard-hgl', download_id: document[:layer_slug_s] },
       class: 'btn btn-default') %>
+      <% elsif !@document.references.nil? && !@document.references.iiif.nil? %>
```

Comment:
I wonder if this could be it's own partial?

Could we potentially move the logic to a helper method too?